app/script/calling/CallingRepository.js

```diff
@@ -1180,7 +1188,18 @@ z.calling.CallingRepository = class CallingRepository {
     return this._createCall(callMessageEntity, this.userRepository.self()).then(callEntity => {
       const mediaType = this._getMediaTypeFromProperties(properties);
       const conversationName = callEntity.conversationEntity.display_name();
-      this.logger.info(`Outgoing '${mediaType}' call in conversation '${conversationName}'`, callEntity);
+      const conversationId = callEntity.conversationEntity.id;
+
+      this.logger.info(
+        {
+          data: {
+            default: [mediaType, conversationName],
+            obfuscated: [mediaType, conversationId],
+          },
+          message: z.util.format_string`Outgoing '${0}' call in conversation '${1}'`,
```

Comment:
Why do you have z.util.format_string in each of these? Only the logger should call them internall, should't it?